packages/errors/src/errorTypes.ts

```diff
@@ -65,4 +65,17 @@ export interface SerializedError {
   message: string
   name: string
   isCypressErr?: boolean
+  tsErrorLocation?: {
```

Comment:
JS errors will have a stack trace that corresponds to the correct line where there's an error. When `ts-node` is handling the parsing of files and there's a syntax error, it will put the failure as the relative path from the project root within the message. That's why the `tsErrorLocation` is added to the error that's caught in the block `if (err.name === 'TSError') {`. Will add comments to explain this

Reply:
Updated w/ comment